moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -154,30 +154,30 @@ void RobotModelLoader::configure(const Options& opt)
           }
 
           param_name = prefix + "max_velocity";
-          double max_velocity;
+          double max_velocity = node_->declare_parameter(param_name, 0.0);
           if (node_->get_parameter(param_name, max_velocity))
           {
             joint_limit[joint_id].has_velocity_limits = true;
             joint_limit[joint_id].max_velocity = max_velocity;
           }
 
           param_name = prefix + "has_velocity_limits";
-          bool has_vel_limits;
+          bool has_vel_limits = node_->declare_parameter(param_name, false);
           if (node_->get_parameter(param_name, has_vel_limits))
           {
             joint_limit[joint_id].has_velocity_limits = has_vel_limits;
           }
```

Comment:
This was a previously-existing issue, not your code, but maybe now is a good time to fix it.

joint_limit[joint_id].has_velocity_limits is already set at L160 so I think you could delete this block

Reply:
If it does not have limits should we set these to default values?

Reply:
I think not. Hopefully @JafarAbdi or @henningkayser  can confirm